src/custom-element.js

```diff
@@ -465,6 +465,12 @@ export function createAmpElementProto(win, name, implementationClass) {
         this.actionQueue_ = null;
       }
     }
+    if (!this.getPlaceholder()) {
```

Comment:
There's a bit of a inconsistency here. The visible placeholder is the _last_ one, but we toggle on and off the first one. See `togglePlaceholder` element. We either need to expand that method to operate on ALL placeholders in the element or we may need to do some form of cleanup, e.g. to remove the superfluous ones. PTAL.